apps/web/src/modules/job/apply/pages/hooks/index.ts

```diff
@@ -0,0 +1,71 @@
+import { GetJobDto, GetResumesDto, ResumeDto, ResumeIdDto } from '@modela/dtos'
+import { useSnackbar } from 'common/context/SnackbarContext'
+import { useErrorHandler } from 'common/hooks/useErrorHandler'
+import { apiClient } from 'common/utils/api'
+import { useRouter } from 'next/router'
+import React from 'react'
+import { useCallback, useEffect, useState } from 'react'
+
+const useResume = () => {
+  const router = useRouter()
+  const { jobId } = router.query
+  const { handleError } = useErrorHandler()
+  const [loading, setLoading] = useState(false)
+  const [jobTitle, setJobTitle] = useState('')
+  const { displaySnackbar } = useSnackbar()
+
+  const [id, setId] = React.useState<number | undefined>()
+
+  const handleSuccess = useCallback(async () => {
+    setLoading(true)
+    try {
+      if (id) {
+        const resumeIdBody: ResumeIdDto = {
+          resumeId: id,
+        }
+        await apiClient.post('jobs/' + jobId + '/apply', resumeIdBody)
+        displaySnackbar('การสมัครเสร็จสิ้น', 'success')
+        router.push('/job', undefined, { shallow: true })
+      } else {
+        displaySnackbar('กรุณาเลือกเรซูเม่', 'error')
+      }
```

Comment:
(Discussion)
Should we change handle missing resumeId from `displaySnackbar` to assign message to `helperText` in TextField instead ?

Reply:
Please see other page such as login, register for reference